src/Http/HttpAbstractions.slnf

```diff
@@ -1,53 +1,54 @@
-﻿{
+{
   "solution": {
```

Comment:
Can you revert this? What changed here?

Reply:
I wish VS made a smaller diff, but do I really have to hand edit it? I added the MapActionSample app.

Reply:
Oh ok